gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/FlowTriggerHandler.java

```diff
@@ -129,6 +131,7 @@ public void handleTriggerEvent(Properties jobProps, DagActionStore.DagAction flo
               leaseObtainedStatus.getEventTimeMillis());
           return;
         }
+        this.failedPersistingLeaseCount.mark();
```

Comment:
shall we also have a meter for success?  or are you planning to compute from (successfullyObtained - failedToRecord/Persist)?

Reply:
good catch there are only logs for this so adding a metric.